hardware_interface/include/hardware_interface/internal/interface_manager.h

```diff
@@ -142,12 +162,23 @@ class InterfaceManager
   /**
    * \brief Get an interface.
    *
-   * Since this class only stores one interface per type, this returns a
-   * pointer to the requested interface type. If the interface type is not
-   * registered, it will return \c NULL.
+   * If this class and its registered sub-managers only have one registered
+   * instance of the requested type, this will be returned. If multiple
+   * instances of the interface type were registered and the type is a
+   * \c ResourceManager, this call will try to combine them into a single
+   * handle. In all other cases, \c nullptr will be returned.
+   *
+   * \note As this instance will only store one instance of each manager type,
+   * the only way multiple interfaces are registered is the registration of
+   * other managers with interfaces of the same type.
```

Comment:
Small language clarification. (Not sure if this suggestion respects line lengths, some reformatting may be needed)

```suggestion
   * If this class and its registered sub-managers only have one registered
   * instance of the requested interface type, this will be returned. If multiple
   * instances of the interface type were registered and the type is a
   * \c ResourceManager, this call will try to combine them into a single
   * handle. In all other cases, \c nullptr will be returned.
   *
   * \note As this class can only store one instance of each interface type,
   * the only way multiple instances of the same interface type can be
    * registered is through the registration of sub-managers.
```

Reply:
Sure, applied in 04bcb42a80da3b7f6e8dc498532723ab8ba00fee